install_config/topics/ocp_osp_provisioning.adoc

```diff
@@ -220,6 +225,40 @@ If Kuryr is enabled, {product-title} services are implemented through OpenStack
 Octavia does not support UDP load balancing. Services that expose UDP ports are not supported.
 ====
 
+[NOTE]
+====
+The `kuryr_openstack_global_namespace` parameter contains a list that defines
+global namespaces.
+By default, only the `default` and `openshift-monitoring` namespaces are included in
+this list.
+====
+
+[IMPORTANT]
```

Comment:
Rather than write this up myself:
1. Remove `[NOTE]` and its block marks, i.e. `====`. 
2. Remove `[IMPORTANT]` and its block marks.
3. At the top of this section, add:
```
[[kuryr-global-namespace-security-groups]]
===== Configuring global namespace access

```
Does that seem okay? Feel free to change the title, too.

Reply:
Oh, and then convert the instructions that follow into an OL.

Reply:
I'm not sure I got this right, I updated it with that I understood, please check. I don't know what OL means here (too many acronysms...). Feel free to push a modification of course

Reply:
OL = ordered list. Will check it out!